### DISCUSSION TITLE
Can't access github copilot

### DISCUSSION THREAD

**Discussion by xiahang123** [community/community#116629] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi, I verified my student identity and got student pack access, but I still can't access github copilot, can anyone help me with that?
It looks great, but when I click **start free trial**, it returns to the same page. Any help will be appreciated.

![Screenshot from 2024-04-03 17-29-01](https://github.com/community/community/assets/88528127/5ea0ff45-fff6-4290-964d-da6d67a75ee5)
![Screenshot from 2024-04-03 17-31-25](https://github.com/community/community/assets/88528127/f6c801ea-ddc4-4750-824f-5883f7fc76c7)
![Screenshot from 2024-03-29 11-57-39](https://github.com/community/community/assets/88528127/3b370904-5d2b-4baa-b975-8cbe4d0d600c)

**Comment by Akash1134** [comment, marked as answer]
Hi @xiahang123 

I have reviewed your account activity and It appears that your access to Copilot was restricted by our automated abuse detection mechanisms. Our systems is constantly learning and we are constantly adjusting to ensure proper abuse detection. 

Just for basic security hygiene; ensure that your credentials are only being used by you, and that you are using Copilot with the [supported](https://docs.github.com/en/copilot/using-github-copilot/getting-started-with-github-copilot) plugins and extensions available. Please note that, scripting against the API is not allowed. This also includes automated/excessive usage and/or programmatic token generations.

I do encourage you to familiarize yourself with our [Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service) and our [GitHub Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) to get better understanding on how to use our services.

To get this sorted, I'd recommend opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by ayim** [comment]
Locking this discussion since it's account related
